presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java

```diff
@@ -110,12 +110,6 @@ public OrcPageSource(
     }
 
     @Override
-    public long getTotalBytes()
-    {
-        return recordReader.getSplitLength();
```

Comment:
After this change `OrcRecordReader::getSplitLength()` is only used in tests, we can remove that too.

Reply:
It's used by `OrcPageSource.getTotalBytes()`

Reply:
Nice catch on these. I'll check with @dain on if he wants to remove them.